src/components/Exif/Exif.tsx

```diff
@@ -0,0 +1,216 @@
+import { useState, useEffect, useCallback } from "react";
+import { Button, Stack, TextInput, Checkbox } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { RenderComponent } from "../UserGuide/UserGuide";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+
+/**
+ * Represents the form values for the Exif component.
+ */
+interface FormValuesType {
+    filePath: string;
+    tag: string;
+    value: string;
+    actionType: string; // "read" or "write"
+}
+
+/**
+ * The Exif component.
+ * @returns The Exif component.
+ */
+const ExifTool = () => {
+    // Component state variables
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution
+    const [allowSave, setAllowSave] = useState(false); // State variable to allow saving of output
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if output has been saved
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
+
+    // Component Constants
+    const title = "ExifTool";
+    const description =
+        "ExifTool is a platform-independent command-line application for reading, writing, and editing metadata in a wide variety of file types.";
+    const steps =
+        "Step 1: Install relevant dependencies from start page\n" +
+        "Step 2: Have an image saved locally and specify file path in the form.\n" +
+        "Step 3: Enter a Metadata tag e.g.Author\n" +
+        "Step 4: Choose to either write or read metadata from the image" +
+        "Step 5: Run the tool!"; // Steps explaining how to run tool
```

Comment:
Simplification of user guide. Also some fixes with punctuation and formatting.

```suggestion
        "Step 1: Specify the file path of an image on your machine.\n" +
        "Step 2: Enter a metadata tag to read or write to.\n" +
        "Step 3: Choose whether to read or write to the specified metadata field.\n" +
        "Step 4: Input a value to write to the specified metadata field.\n" +
        "Step 5: Run the tool!";